packages/nocodb/src/lib/models/Model.ts

```diff
@@ -750,19 +750,29 @@ export default class Model implements TableType {
       },
       null,
       exclude_id && { id: { neq: exclude_id } }
-    ));
+    );
+
+    if (!meta) return true;
+    if (meta?.table_name || undefined == table_name) {
+      if (meta?.project_id == project_id) {
+        return false;
+      }
```

Comment:
Is this if condition really required?  `project_id` will always be the same since we are querying with project_id